docs/user-guide/ze-install-configuring-ze.md

```diff
@@ -0,0 +1,83 @@
+# Configuring Zowe Explorer
+
+:::info Required role: Developer
+:::
+
+Configure Zowe Explorer in the settings file so the extension performs according to your preferences.
+
+To access Zowe Explorer settings:
+
+1. Click the **Settings** icon at the bottom of the **Activity Bar**.
```

Comment:
Should this be renamed to the "Manage" icon for accessibility purposes? Granted, my VS Code setup is configured to use a different layout, but this is the label shown for that icon when hovering:

![image](https://github.com/zowe/docs-site/assets/86500639/c9020574-36f3-451a-b3c6-f9c659fdf5dd)